circuit_maintenance_parser/parsers/apple.py

```diff
@@ -0,0 +1,58 @@
+import email
+import re
+
+from typing import Dict, List
+
+from circuit_maintenance_parser.output import Impact, Status
+from circuit_maintenance_parser.parser import EmailSubjectParser, Text, CircuitImpact
+
+class SubjectParserApple(EmailSubjectParser):
+    def parser_hook(self, raw: bytes):
+        message = email.message_from_string(self.bytes_to_string(raw))
+        return self.parse_subject(message['subject'])
+
+    def parse_subject(self, subject: str) -> List[Dict]:
+        return [{'summary': subject}]
+
+
+class TextParserApple(Text):
+    def parse_text(self, text: str) -> List[Dict]:
+        data = {
+            'circuits': self._circuits(text),
+            'maintenance_id': self._maintenance_id(text),
+            'start': self._start_time(text),
+            'end': self._end_time(text),
+            'status': Status.CONFIRMED, # Have yet to see anything but confirmation.
+            'organizer': 'peering-noc@group.apple.com',
+            'provider': 'apple',
+        }
+        return [data]
+
+    def _circuits(self, text):
+        pattern = r'Peer AS: (\d*)'
+        m = re.search(pattern, text)
+        return [CircuitImpact(
+            circuit_id=f'AS{m.group(1)}',
+            impact=Impact.NO_IMPACT)]
```

Comment:
I guess you would like to mark the circuit state during the maintenance as OUTAGE? NO_IMPACT is supposed for a circuit that it's not being affected during the maintenance period